ui/src/groups/GroupSidebar/ChannelList.tsx

```diff
@@ -45,31 +45,37 @@ interface ChannelListProps {
 
 export function ChannelSorter({ isMobile }: ChannelSorterProps) {
   const { sortFn, sortOptions, setSortFn } = useChannelSort();
+
+  function sortLabel() {
+    switch (sortFn) {
+      case 'Arranged':
+        return 'Arranged Channels';
+      case 'Recent Activity':
+        return 'Recent Activity';
```

Comment:
Could just have the case set as "Recent" here and return "Recent Activity" to avoid the issue I mentioned.

Reply:
Not sure what happened here, good catch.